ec2-cluster/modules/genesis-provisioner/templates/restart-cluster

```diff
@@ -10,6 +10,10 @@ clr='\033[0m'
 READLINK=`readlink $0`
 if [[ "$READLINK" == "" ]]; then DIR="$PWD"; else DIR="$(dirname $READLINK)"; fi
 
+pssh() {
```

Comment:
Why do you need to create such alias? pssh should be already available 🤔

Reply:
I think it is because of Debian: https://askubuntu.com/questions/119232/why-pssh-command-is-not-working